src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
 body {
   margin: 0;
 }
+@import "./variables";
+@import "./calendar";
+@import "./normalize";
+@import "./body";
+$startDay:
```

Comment:
why do you have this variable inside the main file scss ? I think you have pasted it here by mistake